packages/engine/src/worker/runner/javascript/mod.rs

```diff
@@ -872,7 +1262,8 @@ impl<'m> RunnerImpl<'m> {
                     ));
                 };
             }
-            data_type => return Err(Error::FlushType(data_type.clone())), // TODO: More types?
+            data_type => return Err(JSRunnerError::FlushType(data_type.clone())), /* TODO: More
+                                                                                   * types? */
```

Comment:
```suggestion
            // TODO: More types?
            data_type => return Err(JSRunnerError::FlushType(data_type.clone())),
```

Reply:
Addressed in https://github.com/hashintel/hash/pull/414/commits/02af15786b564ada4f9992194281ff0a3281afb9